.github/actions/diff-uploader/action.yml

```
@@ -0,0 +1,15 @@
+name: 'Diff Uploader'
+description: 'Creates a diff between the commit and the latest Quilt Mappings version'
```

would be nice if this specified that it's a code diff, it seems pointless at a glance otherwise
```suggestion
description: 'Creates a diff between the decompiled Minecraft code of the commit and the latest Quilt Mappings version'
```